src/components/Article/ArticleController.tsx

```diff
@@ -52,8 +41,10 @@ const ArticleController = (props: Props) => {
     question: '',
     answer: '',
     newCategory: '',
+    searchCriteria: '',
```

Comment:
searchCriteria can be moved into it's own state

Reply:
created searchCriteria state